src/plugins/tts/plugin.tts.js

```diff
@@ -158,18 +158,35 @@ BookReader.prototype.initNavbar = (function (super_) {
 
       $el.find('.BRcontrols').prepend(this.refs.$BRReadAloudToolbar);
 
+      const renderVoiceOption = (language) => {
+        return language.map(voice =>
+          `<option value="${voice.voiceURI}">${voice.lang} - ${voice.name}</option>`);
+      };
+
       const renderVoicesMenu = (voicesMenu) => {
         voicesMenu.empty();
+
+        const bookLanguages = this.ttsEngine.getVoices().filter(v => v.lang.startsWith('en')).sort((a,b) => {
+          if (a.lang > b.lang) return 1;
+          if (a.lang < b.lang) return -1;
+          return 0;
+        });
```

Comment:
Put the sorting function in a helper, so that we can avoid the duplicate code. You can also simplify this by using `localeCompare` to compare the strings: `(a, b) => a.lang.localeCompare(b.lang)`. And +1 sorting both looks much better!

Let's also sort by the voice name! We can do that by sorting over `${voice.lang} - ${voice.name}` instead of just `voice.lang`.

Reply:
about the sorting part: 
- I made a sorting helper, which sorts normally
- However, little confused as to how to sort here, I did sort like so, `voice.lang` - `voice.name`


```
const sortedOrder = () => {
        return (a,b) => a.lang.localeCompare(b.name);
      };
```
<img width="439" alt="Screen Shot 2022-05-20 at 2 35 49 PM" src="https://user-images.githubusercontent.com/52472550/169615678-fd9173ec-516c-4871-a81c-c47e5897b2c6.png">

Result comes out as so.

Reply:
Ah, I meant sort the string:

```
`${a.lang} - ${a.name}`.localeCompare(`${b.lang} - ${b.name}`)
```

Reply:
This will let the sort take into account both the language code, locale, and name :+1:

Reply:
@cdrini  

doing so didn't really change the result. The voice option looks similar to the first PR picture I did. 
I'll do a PR, so you can see, what I may have done wrong.  :)

Reply:
Hmm, it's working correctly for me now after the changes; can you send a screenshot?

Reply:
<img width="380" alt="Screen Shot 2022-05-20 at 3 56 35 PM" src="https://user-images.githubusercontent.com/52472550/169622393-b6ebaa5a-eb6e-41fa-92b6-079613cbd21c.png">

This is same ordering as the first PR.

Reply:
No that looks correct! Note how `en-US - Google US English` is now before `Samantha` :)

Reply:
@ ooops, really missed that part of the order. 
Guess I was staring at those options for long. I couldnt catch it haha.

Reply:
@cdrini so looks good to merge?

Reply:
Oh you'll need to make the changes in the other comments :) Those are just code suggestions; I haven't actually modified the code! Then I'll do some cross browser testing, and we should be good!

Reply:
Oh whoops you already have! My apologies I think I was looking at an old diff. Let me do another pass!

Reply:
yea, done with all the changes